HW04/ekaterina_balabanovich/lib/homework.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class Homework
+  attr_reader :title, :description
+  attr_accessor :answers
+
+  def initialize(title, description)
+    @title = title
+    @description = description
+    @answers = Hash.new
+  end
+
+  def solve(answer, student)
+    answers[student] = answer
+  end
```

Comment:
This is not a good practice to use Hash for this kind of things https://refactoring.guru/ru/smells/primitive-obsession would be better to create a separated class or just a new `Struct` https://ruby-doc.org/core-2.5.0/Struct.html
```suggestion
    @answers = []
  end
  
  Answer = Struct.new(:student, :answer)

  def solve(answer, student)
    answers << Answer.new(student, answer)
  end

```